### PR TITLE
Simple documentation + saved docs

### PR DESCRIPTION
**Pull request by npadmana**
Simple documentation. Mostly for us while using this routine.

I'm also including the generated documentation, so that it can be hosted on github.

### CODE REVIEW

**Comment by npadmana** [comment]
@ronawho -- care to look over this? You might find it easiest to actually just read the website.

File: src/DistributedFFT.chpl [hunk 1]
```diff
@@ -80,19 +133,50 @@ prototype module DistributedFFT {
...
+      :arg dest: Output array, transposed [YXZ]
+      :arg signOrKind: Sign (for DFT), or kind (for R2R) of transform.
+
+      Refer to the FFTW documentation for the different
+      signs/kinds.
+
+      Note that if ``ftType`` is ``FFTtype.R2R``, then
+      the ``signOrKind`` argument must be a 3 element array
+      (the type for each dimension separately).
+  ***/
```
Comment (ronawho) [inline src/DistributedFFT.chpl]: I missed this before but maybe update array/domain names to match the performant/naive functions (src->Src, dest -> Dest, DestDom->DstDom)

Comment (npadmana) [reply src/DistributedFFT.chpl]: Fixed....

File: src/DistributedFFT.chpl [hunk 2]
```diff
@@ -247,13 +338,28 @@ prototype module DistributedFFT {
     }
   }
 
+  pragma "no doc"
   iter offset(r: range) { halt("Serial offset not implemented"); }
```
[the remark this answers is not in the document]
Comment (npadmana) [reply src/DistributedFFT.chpl]: Fixed....

**Pull request merged**